api/src/service/domain/workflow/workflowitem_create.ts

```diff
@@ -199,6 +203,16 @@ export async function createWorkflowitem(
     if (Result.isErr(result)) {
       return result;
     }
+    const { document } = result as WorkflowitemDocumentUploaded.Event;
+    // document should be private
+    if (minioEndPoint) {
+      await repository.uploadDocument(document);
+      const eventData = {...result, document: {...document, base64: "", url: hostPort}};
+      documentUploadedEvents.push(eventData);
+    } else {
+      documentUploadedEvents.push(result);
+
+    }
     documentUploadedEvents.push(result);
```

Comment:
@galethil Is any reason to push to `documentUploadedEvents` (always) twice?

Reply:
Good catch! I removed the duplication.

Reply:
Nice